pkgs/applications/audio/elisa/default.nix

```diff
@@ -12,7 +12,7 @@ mkDerivation rec {
   src = fetchFromGitHub {
     owner  = "KDE";
     repo   = "elisa";
-    rev    = version;
+    rev    = "v${version}";
```

Comment:
That's not the issue: `rev = version;` works fine. It fails at installPhase because it fails to remove a non-empty dir.

Reply:
Hmm, I was building locally and It failed with 404 without this. I also noticed that this builds fine without any of the recent KDE upgrades...